src/components/dialogs/line-creation-dialog.js

```diff
@@ -277,9 +278,11 @@ const LineCreationDialog = ({
                 editData ? true : false,
                 editData ? editData.uuid : undefined,
                 connectivity1?.connectionName?.id ?? null,
-                connectivity1?.connectionDirection?.id ?? 'UNDEFINED',
+                connectivity1?.connectionDirection?.id ??
```

Comment:
rename to connectivityOne maybe better

Reply:
I've just replaced the string 'UNDEFINED' by a constant, so I will not rename connectivity1, connectivity2.
And also shuntConductance1, shuntConductance2, shuntSusceptance1, shuntSusceptance2, ...